src/app/data/components/record-list/record-list.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Attributes } from 'src/app/base/components/record-display/record-display.helper';
+import { ColumnData, RecordListOptions } from './record-list-helper';
+
+@Component({
+  selector: 'kern-record-list',
+  templateUrl: './record-list.component.html',
+  styleUrls: ['./record-list.component.scss']
+})
+export class RecordListComponent implements OnInit {
+
+  @Input() recordList: any[];
+  @Input() attributes: Attributes;
+  @Input() recordListOptions: RecordListOptions;
+
+  @Output() indexHuddleData = new EventEmitter<number>();
```

Comment:
the name suggest to me that this is a number not an eventemmitter for a number. Maybe to generalize it a bit `recordClicked`?